src/elements/kw-blank-project-modal/kw-blank-project-modal.html

```diff
@@ -0,0 +1,318 @@
+                    width: 205px;
+                    height: 205px;
+                    margin-bottom: 38px;
+                }
+                .mode-options > a:hover {
+                    transform: none;
+                    opacity: 1;
+                }
+                .mode-options > a:hover .mode-button {
+                    background-color: #7ac943;
+                }
+                .mode-options > a:active .mode-button {
+                    background-color: #62b22a;
+                }
+                .mode-button {
+                    margin: 0 6px;
+                }
+                .close-button-display {
+                    display: none;
+                }
+                .close-button-mobile {
+                    @apply --kano-button;
+                    display: inline-block;
+                    align-self: flex-end;
+                    background-color: black;
+                    position: relative;
+                    top: -2px;
+                    right: -5px;
+                    border: 0;
+                    border-radius: 25px;
+                    text-transform: none;
+                    font-size: 16px;
+                    margin: 5px 0 20px !important;
+                    padding: 8px 15px !important;
+                }
+            }
+        </style>
+        <iron-image id="overlay" class="background-image" src="/assets/projects/modes/modal-bg-s.svg" sizing="contain" alt="Modal overlay" preload fade>
+        </iron-image>
+        <div class="dialog-wrapper">
+            <button class="close-button-display" type="button" on-click="close">Close <div class="close-symbol">&#x00D7;</div></button>
+            <paper-dialog id="modal" opened="{{opened}}">
+                <div class="container">
+                    <button class="close-button-mobile" type="button" on-click="close">Close <div class="close-symbol">&#x00D7;</div></button>
+                    <h5>What do you want to build with?</h5>
+                    <div class="mode-options">
+                        <a href="https://apps.kano.me/make">
+                            <div class="mode">
+                                <iron-image src="/assets/projects/modes/canvas-blank.png" sizing="contain" preload fade alt="Start a new project"></iron-image>
+                                <button type="button" class="mode-button">blank canvas</button>
+                            </div>
+                        </a>
+                        <a href="https://apps.kano.me/make?mode=lightboard">
+                            <div class="mode">
+                                <iron-image src="/assets/projects/modes/canvas-pixel.png" sizing="cover" preload fade alt="Start a new lightboard project"></iron-image>
+                                <button type="button" class="mode-button">pixel kit</button>
+                            </div>
+                        </a>
+                        <a href="https://apps.kano.me/make?mode=camera">
```

Comment:
/make doesn't exist anymore. you can put /?mode=camera directly, it will avoid a redirect